src/provider/sockjs-provider.tsx

```diff
@@ -3,21 +3,19 @@ import { useParams } from 'react-router-dom';
 import SockJS from 'sockjs-client';
 import { CompatClient, Stomp } from '@stomp/stompjs';
 import { getLocalStorage } from '@/utils/storage';
-import { ParticipantSummary, Step } from '@/types/study-room.types';
+import { ParticipantSummary, Step, StepInfo } from '@/types/study-room.types';
 
 interface SockJSContextType {
   curParticipants: ParticipantSummary[];
-  curStep: Step;
-  curUpdateAt: string;
+  curStepInfo: StepInfo | null;
   goToNextStep: (step: Step) => void;
 }
 
 export const SockJSContext = createContext<SockJSContextType | null>(null);
 
 const SockJSProvider = ({ children }: PropsWithChildren) => {
   const [curParticipants, setCurParticipants] = useState<ParticipantSummary[]>([]);
-  const [curStep, setCurStep] = useState<Step>('PLANNING');
-  const [curUpdateAt, setCurUpdateAt] = useState('');
+  const [curStepInfo, setCurStepInfo] = useState<StepInfo | null>(null);
```

Comment:
축약하지 않고 그대로 표현해주면 더 좋을 것 같아요~
```suggestion
  const [currentStepInfo, setCurrentStepInfo] = useState<StepInfo | null>(null);
```

Reply:
짚어주신 덕에 다른 프론트엔드 개발자와 상의해서 current -> cur로 축약하여 사용하는 걸로 결정하고 코드 컨벤션에 추가하기로 했습니다~!

Reply:
네! 컨벤션만 합의되면 축약해서 사용해도 좋습니다~